mne/beamformer/_dics.py

```diff
@@ -117,6 +117,8 @@ def make_dics(info, forward, csd, reg=0.05, noise_csd=None, label=None,
                 The normalization of the weights.
             'src_type' : str
                 Type of source space.
+            'source_nn' : ndarray, shape (n_sources, 3)
+                For each source location, the surface normal.
```

Comment:
I think it has to be stored in the filter so that when you pass the object in `apply_*` you have this information. It's also consistent with how it's done for LCMV.